2009-12-02  Stepan Kasal  <skasal@redhat.com>

	* add the extra symbols, even though DEBUGGING is not defined

diff -ur perl-5.10.1.orig/hv.c perl-5.10.1/hv.c
--- perl-5.10.1.orig/hv.c	2009-06-10 14:36:34.000000000 +0200
+++ perl-5.10.1/hv.c	2009-12-02 15:05:07.000000000 +0100
@@ -2926,7 +2926,7 @@
 =cut
 */
 
-#ifdef DEBUGGING
+/* #ifdef DEBUGGING */
 
 void
 Perl_hv_assert(pTHX_ HV *hv)
@@ -2991,7 +2991,7 @@
     HvEITER_set(hv, eiter);
 }
 
-#endif
+/* #endif */
 
 /*
  * Local variables:
diff -ur perl-5.10.1.orig/makedef.pl perl-5.10.1/makedef.pl
--- perl-5.10.1.orig/makedef.pl	2009-12-02 14:42:12.000000000 +0100
+++ perl-5.10.1/makedef.pl	2009-12-02 14:42:04.000000000 +0100
@@ -617,7 +617,7 @@
 			)];
 }
 
-unless ($define{'DEBUGGING'}) {
+unless (1 || $define{'DEBUGGING'}) {
     skip_symbols [qw(
 		    Perl_deb_growlevel
 		    Perl_debop
diff -ur perl-5.10.1.orig/pad.c perl-5.10.1/pad.c
--- perl-5.10.1.orig/pad.c	2009-04-22 23:43:43.000000000 +0200
+++ perl-5.10.1/pad.c	2009-12-02 14:39:21.000000000 +0100
@@ -912,7 +912,7 @@
 }
 
 
-#ifdef DEBUGGING
+/* #ifdef DEBUGGING */
 /*
 =for apidoc pad_sv
 
@@ -963,7 +963,7 @@
     );
     PL_curpad[po] = sv;
 }
-#endif
+/* #endif */
 
 
 
diff -ur perl-5.10.1.orig/perl.h perl-5.10.1/perl.h
--- perl-5.10.1.orig/perl.h	2009-07-06 13:18:58.000000000 +0200
+++ perl-5.10.1/perl.h	2009-12-02 15:06:44.000000000 +0100
@@ -4457,7 +4457,7 @@
 EXTCONST unsigned char PL_freq[];
 #endif
 
-#ifdef DEBUGGING
+/* #ifdef DEBUGGING */
 #ifdef DOINIT
 EXTCONST char* const PL_block_type[] = {
 	"NULL",
@@ -4473,7 +4473,7 @@
 #else
 EXTCONST char* PL_block_type[];
 #endif
-#endif
+/* #endif */
 
 /* These are all the compile time options that affect binary compatibility.
    Other compile time options that are binary compatible are in perl.c
diff -ur perl-5.10.1.orig/perlvars.h perl-5.10.1/perlvars.h
--- perl-5.10.1.orig/perlvars.h	2009-02-12 23:58:17.000000000 +0100
+++ perl-5.10.1/perlvars.h	2009-12-02 15:10:39.000000000 +0100
@@ -112,9 +112,9 @@
 PERLVARI(Gsig_trapped, int, 0)
 #endif
 
-#ifdef DEBUGGING
+/* #ifdef DEBUGGING */
 PERLVAR(Gwatch_pvx, char*)
-#endif
+/* #endif */
 
 #ifdef PERL_GLOBAL_STRUCT 
 PERLVAR(Gppaddr, Perl_ppaddr_t*) /* or opcode.h */
--- perl-5.10.1/proto.h.kasal	2009-12-02 15:18:39.000000000 +0100
+++ perl-5.10.1/proto.h	2009-12-02 15:29:51.000000000 +0100
@@ -6068,13 +6068,13 @@
 #define PERL_ARGS_ASSERT_PAD_CHECK_DUP	\
 	assert(name); assert(ourstash)
 
-#ifdef DEBUGGING
+/* #ifdef DEBUGGING */
 PERL_CALLCONV void	Perl_pad_setsv(pTHX_ PADOFFSET po, SV* sv)
 			__attribute__nonnull__(pTHX_2);
 #define PERL_ARGS_ASSERT_PAD_SETSV	\
 	assert(sv)
 
-#endif
+/* #endif */
 PERL_CALLCONV void	Perl_pad_block_start(pTHX_ int full);
 PERL_CALLCONV void	Perl_pad_tidy(pTHX_ padtidy_type type);
 PERL_CALLCONV void	Perl_do_dump_pad(pTHX_ I32 level, PerlIO *file, PADLIST *padlist, int full)
